org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/properties/PropertiesConverter.java

```diff
@@ -0,0 +1,249 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+package org.osgi.test.junit5.properties;
+
+import static org.osgi.test.common.annotation.Property.Type.Scalar;
```

Comment:
Seems like this can be confused with the `import org.osgi.test.common.annotation.Property.Scalar;` below. Two things called `Scalar`. Perhaps we should not statically import this and just use `Type.Scalar`.